### PR TITLE
[JENKINS-62264] User with MANAGE permissions can access node monitoring

### PR DESCRIPTION
**Pull request by amuniz**
See [JENKINS-62264](https://issues.jenkins-ci.org/browse/JENKINS-62264).

### Proposed changelog entries

* JENKINS-62264: Users with MANAGE permission can configure Node Monitoring.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- [x] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [x] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by fcojfernandez — APPROVED** [review]

**Review by alecharp — APPROVED** [review]
Everything seems good. Thanks!

**Review by aHenryJard — APPROVED** [review]
Looks good to me.

**Review by EstherAF — APPROVED** [review]
I've tested and everything seems good.

**Review by varyvol — APPROVED** [review]

**Review by MRamonLeon — APPROVED** [review]

**Comment by timja** [comment]
We'll merge this in 24 hours if no negative feedback

**Review by markjacksonfishing — APPROVED** [review]

**Comment by daniel-beck** [comment]
Needs merge conflict resolution after #4531.

File: core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly [hunk 1]
```diff
@@ -34,8 +34,9 @@ THE SOFTWARE.
       <l:task href="${rootURL}/" icon="icon-up icon-md" title="${%Back to Dashboard}"/>
       <l:task href="${rootURL}/manage" icon="icon-gear2 icon-md" permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%Manage Jenkins}"/>
       <l:task href="new" icon="icon-new-computer icon-md" permission="${createPermission}" title="${%New Node}"/>
-      <l:task href="${rootURL}/configureClouds" icon="icon-health-40to59 icon-md" permission="${app.ADMINISTER}" title="${%Configure Clouds}"/>
-      <l:task href="configure" icon="icon-gear2 icon-md" permission="${app.MANAGE}" title="${%Node Monitoring}"/>
+      <l:task href="${rootURL}/configureClouds" icon="icon-health-40to59 icon-md" permission="${app.SYSTEM_READ}"
+              title="${app.hasPermission(app.ADMINISTER) ? '%Configure Clouds' : '%View Clouds'}"/>
```
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly:38]: Can Overall/Manage users not configure clouds?

Comment (amuniz) [reply core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly:38]: Not sure... to configure a cloud you need to install a cloud implementation plugin, but for now Overall/Manage is not allowed to install plugins. I'd prefer to delay this decision a bit.

Comment (daniel-beck) [reply core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly:38]: No worries, was just confused, that looks like something Manage should be able to and would be very desirable (assuming it's safe).

**Review by daniel-beck — COMMENTED** [review]
Would consider this incomplete while `ComputerSet#doUpdateNow` remains inaccessible; a typical use case seems to be to adjust limits and then apply them to see the effect, that doesn't seem possible right now.

File: core/src/main/resources/hudson/model/ComputerSet/configure.jelly [hunk 2]
```diff
@@ -39,12 +40,12 @@ THE SOFTWARE.
                           descriptors="${it.nodeMonitorDescriptors}"
                           instances="${it.nonIgnoredMonitors}" />
 
-        <l:isAdmin>
+        <j:if test="${h.hasPermission(app.MANAGE)}">
```
[the remark this answers is not in the document]
Comment (amuniz) [reply core/src/main/resources/hudson/model/ComputerSet/configure.jelly]: I didn't know that, thanks! Changed.

**Comment by amuniz** [comment]
> Would consider this incomplete while ComputerSet#doUpdateNow remains inaccessible

@daniel-beck done.

**Review by daniel-beck — APPROVED** [review]
Thanks for the amendment. Untested but looks good.

**Review by Wadeck — APPROVED** [review]
Thanks for the manual testing @EstherAF :)
Thanks for the test @amuniz :)

Code looks good, 🐝

**Pull request merged**